AntonBabunin/src/main/java/ua/artcode/market/creator/TerminalCreator.java

```diff
@@ -1,24 +1,29 @@
 package ua.artcode.market.creator;
 
-
 import ua.artcode.market.bill.Bill;
 import ua.artcode.market.salesman.Salesman;
 import ua.artcode.market.terminal.Terminal;
 
 public class TerminalCreator {
```

Comment:
implement interface

Reply:
Why here? It implemented in Terminal and Salesman.

Reply:
Please attach your uml diagram

Reply:
How I can atach uml diagram?